ui/web/app.py

```diff
@@ -0,0 +1,49 @@
+from typing import Annotated
+from fastapi import FastAPI, Request, File, Form, UploadFile
+from fastapi.templating import Jinja2Templates
+from fastapi.staticfiles import StaticFiles
+from domain.processor import Processor
+import uuid
+import shutil
+
+processor = Processor()
+
+app = FastAPI()
+
+templates = Jinja2Templates(directory="ui/web/frontend/dist")
+
+app.mount("/assets", StaticFiles(directory="ui/web/frontend/dist/assets"), name="assets")
+app.mount("/static", StaticFiles(directory="ui/web/frontend/dist/static"), name="static")
+app.mount("/data", StaticFiles(directory="data"), name="data")
+
+@app.get("/_info")
+def info():
+    return {"status": "ok"}
+
+@app.post("/api/process")
+def info(
+    request: Request,
+    text: Annotated[str, Form()],
+    image: Annotated[UploadFile, File()],
+):
+    data = processor.make_data()
+
+    processor.set_text_and_speaker(data, text, 'en_0')
```

Comment:
It is advisable to incorporate voice selection feature in the future (male/female and voice tone) en_01, en_02, ... en_100